docs/crio.conf.d.5.md

```diff
@@ -0,0 +1,25 @@
+% crio.conf.d(5)
+
+# NAME
+crio.conf.d - directory for drop-in configuration files
+
+# DESCRIPTION
+Additionally to configuration in crio.conf(5) CRI-O allows to drop configuration
+snippets into the crio.conf.d directory. The default directory is /etc/crio/crio.conf.d/.
+The path can be change via CRIO's **--config-dir** command line option.
+
+# CONFIGURATION PRECEDENCE
+When existing, the main configuration file (/etc/crio/crio.conf by default) is
+read before any file in the configuration directory (/etc/crio/crio.conf.d).
+Settings in that file have the lowest precedence.
+
+Files in the configuration directory are sorted by name in lexical order and
+applied in that order. If multiple configuration files specify the same
+configuration option. The setting specified in file sorted last takes
+precedence over any other value. That is, if two files 00-default.conf and
+10-custom.conf exist in crio.conf.d and both specify different values for a
+certain configuration option. The value from 10-custom.conf will be applied.
```

Comment:
suggest "That is, if two files 00-default.conf and 10-custom.conf exist" -> "That is if both 00-default.conf and 10-custom.conf exist"

Also "option. The" -> "option, the"

Reply:
done